libmapvxl.h

```diff
@@ -1,27 +1,30 @@
 #ifndef LIBMAPVXL_H
-#define LIBMAPVXL_H
-
-#include <stddef.h>
-
-#define DEFAULT_COLOR 0xFF674028
-
-typedef struct {
-    unsigned short MAP_X_MAX;
-    unsigned short MAP_Y_MAX;
-    unsigned short MAP_Z_MAX;
-    unsigned char ***blocks;
-    unsigned int ***color;
-} MapVxl;
-
-void mapvxlCreate(MapVxl *map, int maxX, int maxY, int maxZ);
-void mapvxlFree(MapVxl *map);
-void mapvxlLoadVXL(MapVxl *map, unsigned char *v);
-unsigned char mapvxlIsSurface(MapVxl *map, int x, int y, int z);
-void mapvxlSetColor(MapVxl *map, int x, int y, int z, unsigned int c);
-size_t mapvxlWriteMap(MapVxl *map, unsigned char *mapOut);
-unsigned int mapvxlGetColor(MapVxl *map, int x, int y, int z);
-void mapvxlSetAir(MapVxl *map, int x, int y, int z);
-unsigned char mapvxlIsSolid(MapVxl *map, int x, int y, int z);
-unsigned char mapvxlFindTopBlock(MapVxl *map, int x, int y);
-
+	#define LIBMAPVXL_H
```

Comment:
They should not be indented by the ifndef

Reply:
Fixed.